creme/creme_core/tests/models/test_history.py

```diff
@@ -239,20 +239,21 @@ def test_edition02(self):
             f'like "Evangelion" or "Fushigi no umi no Nadia".'
         )
         creation_date = date(year=1984, month=12, day=24)
-        response = self.client.post(
-            gainax.get_edit_absolute_url(),
-            follow=True,
-            data={
-                'user':           user.id,
-                'name':           name,
-                'phone':          phone,
-                'email':          email,
-                'description':    description,
-                'sector':         sector02.id,
-                'creation_date':  creation_date,
-                'subject_to_vat': True,
-            },
-        )
+        with self.assertNumQueries(51):
```

Comment:
You can remove this assertNumQueries (it check we whole form, so it will fail every time a new field adds a query, of another field remove a query, even if it's not related to History).

Reply:
oh I understand, I remove it